src/content-tags-drawer/tags-sidebar/TagsSidebarBody.jsx

```diff
@@ -0,0 +1,109 @@
+import { useState, useMemo } from 'react';
```

Comment:
Missing typing checking
```suggestion
// @ts-check
import { useState, useMemo } from 'react';
```

Reply:
Updated here https://github.com/openedx/frontend-app-course-authoring/pull/852/commits/1d505dad132b3194ae3326e4b59e01b69672e531